### PR TITLE
vp-1254_Format_address_in_Selected_address_field

### PR DESCRIPTION
**Pull request by AliveMen**
Change address parts sequence

### CODE REVIEW

**Comment by AliveMen** [comment]
2020-01-27 09 42-Electronics - Checkout - Google Chrome

File: assets/js/common-components/address.js [hunk 1]
```diff
@@ -75,14 +75,13 @@ storefrontApp.component('vcAddress', {
...
-            stringifiedAddress += address.city + ' ';
+            var stringifiedAddress = address.organization ? address.organization + ', ' : '';
+            stringifiedAddress += address.line2 ? address.line2 + ' ' : '';
             stringifiedAddress += address.line1 + ', ';
-            stringifiedAddress += address.line2 ? address.line2 : '';
-            stringifiedAddress += address.postalCode;
+            stringifiedAddress += address.city + ' ';
+            stringifiedAddress += address.regionName ? address.regionName + ', ' : '';
+            stringifiedAddress += address.postalCode + ', ';
+            stringifiedAddress += address.countryName + ', ';
```
Comment (yecli) [inline assets/js/common-components/address.js]: Seems ', ' after countyName is redundant

File: assets/js/common-components/address.js [hunk 2]
```diff
@@ -75,14 +75,13 @@ storefrontApp.component('vcAddress', {
...
-            stringifiedAddress += address.organization ? address.organization + ', ' : '';
-            stringifiedAddress += address.countryName + ', ';
-            stringifiedAddress += address.regionName ? address.regionName + ', ' : '';
-            stringifiedAddress += address.city + ' ';
+            var stringifiedAddress = address.organization ? address.organization + ', ' : '';
+            stringifiedAddress += address.line2 ? address.line2 + ' ' : '';
             stringifiedAddress += address.line1 + ', ';
-            stringifiedAddress += address.line2 ? address.line2 : '';
-            stringifiedAddress += address.postalCode;
+            stringifiedAddress += address.city + ' ';
```
Comment (yecli) [inline assets/js/common-components/address.js]: Probably you want to add comma after the city.

File: assets/js/common-components/address.js (hunk 1 above)
Comment (AliveMen) [reply assets/js/common-components/address.js]: deleted

File: assets/js/common-components/address.js (hunk 2 above)
Comment (AliveMen) [reply assets/js/common-components/address.js]: added

**Pull request merged**